package.json

```diff
@@ -4,14 +4,16 @@
   "description": "Atomic Flux with hot reloading",
   "main": "lib/index.js",
   "scripts": {
-    "browser": "scripts/browser",
-    "build": "scripts/build",
-    "clean": "scripts/clean",
-    "lint": "scripts/lint",
-    "prepublish": "scripts/prepublish",
```

Comment:
I'm still not convinced we need to remove `prepublish`.
I want installing from Git to “just work”. That's what `prepublish` does if I'm not mistaken.